modules/katello_activation_key.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2017, Andrew Kofink <ajkofink@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: katello_activation_key
+short_description: Create and Manage Katello activation keys
+description:
+    - Create and Manage Katello activation keys
+author: "Andrew Kofink (@akofink)"
+requirements:
+    - "nailgun >= 0.28.0"
+    - "python >= 2.6"
+options:
+    server_url:
+        description:
+            - URL of Foreman server
+        required: true
+    username:
+        description:
+            - Username on Foreman server
+        required: true
+    password:
+        description:
+            - Password for user accessing Foreman server
+        required: true
+    verify_ssl:
+        description:
+            - Verify SSL of the Foreman server
+        required: false
+    name:
+        description:
+            - Name of the activation key
+        required: true
+    organization:
+        description:
+            - Organization label that the activation key is in
+        required: false
+    lifecycle_environment:
+        description:
+            - Name of the lifecycle environment
+        required: false
+    content_view:
+        description:
+            - Name of the content view
+        required: false
+'''
+
+EXAMPLES = '''
+- name: "Create katello client activation key"
+  katello_activation_key:
+    username: "admin"
+    password: "admin"
+    server_url: "https://foreman.example.com"
+    name: "Katello Clients"
+    organization: Default Organization
+    lifecycle_environment: "Library"
+    content_view:
+      - name: 'client content view'
```

Comment:
This example does not meet the docs, I feel like it should just be `content_view: client content view'`